config/devspace/kustomization.yaml

```diff
@@ -0,0 +1,54 @@
+resources:
+- ../default
+
+images:
+- name: quay.io/validator-labs/vsphere-validator:latest
```

Comment:
I don't think we need ":latest" here.

Reply:
true, but fyi every single `devspace/kustomization.yaml` we have specify the `:latest` tag. i dont think its a big deal but can remove it if its really a blocker

Reply:
Agreed that this change is not necessary.